scripts/check_index

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env ruby
+
+require "json-schema"
+require "open3"
+
+INDEX_PATH = "index.json"
+SCHEMA_PATH = "schema.json"
+RBIS_PATH = "rbi/annotations"
+
+class IndexValidator
+  def self.validate!(index_path: INDEX_PATH, schema_path: SCHEMA_PATH, rbis_path: RBIS_PATH)
+    $stderr.puts("Checking index...\n\n")
+
+    validator = new(index_path: index_path, schema_path: schema_path, rbis_path: rbis_path)
+    exit(1) unless validator.validate!
+
+    $stderr.puts("No errors, good job!")
+  end
+
+  def initialize(index_path:, schema_path:, rbis_path:)
+    @index_path = index_path
+    @schema_path = schema_path
+    @rbis_path = rbis_path
+
+    @index_json = load_json(index_path)
+    @schema_json = load_json(schema_path)
+  end
+
+  def validate!
+    success = true
+    success = false unless check_against_schema!
+    success = false unless check_missing_index_entries!
+    success = false unless check_missing_annotations!
+    success = false unless check_json_format!
+    success
+  end
+
+  private
+
+  def check_against_schema!
+    JSON::Validator.validate!(@schema_json, @index_json)
+    return true
```

Comment:
Redundant `return`?
We're returning `true` in `check_json_format!`  so the style also doesn't match.